project_management_system/src/app/app.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { LanguageService } from './services/language.service';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
+})
+export class AppComponent {
+  title = 'project-management-system';
```

Comment:
It better to use modifiers and typing for each property or method. It leads to better readability of code.